lib/liquid/swap/Claim.ts

```diff
@@ -149,15 +172,16 @@ export const constructClaimTransaction = (
       pset.addOutput(
         new CreatorOutput(
           network.assetHash,
-          0,
+          // TODO: figure out flakiness with blinding 0 amount outputs
+          1,
```

Comment:
Only when blinding 0 sat outputs which is needed because spending a blinded input requires at least one blinded output
The bug has always been there. I just noticed it now because there are quite some integration tests for the covenant claim path which has to be unblinded